Dockerfile

```diff
@@ -1,7 +1,12 @@
 FROM g4s8/artipie-base:latest as build_jar
 ARG version="1.0-SNAPSHOT"
 WORKDIR /jar
+
+# Download dependencies
 COPY pom.xml ./
+RUN mvn verify clean --fail-never
```

Comment:
@Sammers21 I don't think we need to use `verify` here - it'll collect all dependencies for `test` phase and all Maven plugins to run goals required for `verify`. It would be more correct to use some `dependencies` goal to collect dependencies

Reply:
@g4s8, dependency:resolve dependency:resolve-plugins?

Reply:
@Sammers21 yes, I think it would be enough